apps/react-18-tests-v9/package.json

```diff
@@ -8,6 +8,7 @@
     "type-check:integration": "tsc -p tsconfig.react-18.json --noEmit",
     "lint": "ESLINT_USE_FLAT_CONFIG=false eslint --ext .js,.ts,.tsx ./src",
     "test": "jest --passWithNoTests",
+    "test:integration": "jest --passWithNoTests -c jest-react-18.config.js",
```

Comment:
similarily to v8 we run in updateSnapshot mode to mitigate false positives from `react-test-renderer`